toc/working-groups/foundational-infrastructure.md

```diff
@@ -148,9 +148,9 @@ areas:
     github: kimago
   - name: Ryan Wittrup
     github: ryanwittrup
-  reviewers:
   - name: Kevin Markwardt
     github: kmarkwardt-vmware
+  reviewers:
```

Comment:
Check is failing:

```
Failed validating 'type' in schema['properties']['areas']['items']['properties']['reviewers']:
    {'items': {'$ref': '#/$defs/githubUser'}, 'type': 'array'}
```

You probably want to make this an array.   e.g. `reviewers: []`

Reply:
Thanks for the update, I missed that.  All fixed now. Thanks